hq/README.md

```diff
@@ -80,3 +80,34 @@ It is possible to run the application and setup nginx as a proxy; this will make
 
 6. Now when you run the project, it will also be accessible via https://security-hq.local.dev-gutools.co.uk/
 
+### Working with CSS and JS
+
+Security HQ uses [Prettier](https://prettier.io), an opinionated code formatter. This can be run as a pre-commit hook, to format all of the stages changes in the CSS and JS.
+
+1. You will need to install [Yarn](https://yarnpkg.com) to handle the dependencies:
+  > *Linux:* Instructions can be found here: https://yarnpkg.com/lang/en/docs/install/#linux-tab
+  >
+  > *Mac OSX:* `$ brew install yarn`
+
+2. Install all the dependencies:
+
+`$ yarn`
+
+3. Run the setup for the dependencies:
+
+`$ yarn run setup`
```

Comment:
that bit didn't work for me.  Am i missing something ?

Reply:
Nope, I deleted that script but forgot to remove the mention in the readme 😅 

Will fix it now, cheers.